src/webpubsub/azext_webpubsub/_params.py

```diff
@@ -33,12 +32,12 @@ def load_arguments(self, _):
         c.argument('webpubsub_name', webpubsub_name_type, options_list=['--name', '-n'])
 
     with self.argument_context('webpubsub create') as c:
-        c.argument('sku', arg_type=get_enum_type(SKU_TYPE), help='The sku name of the signalr service.')
-        c.argument('unit_count', help='The number of signalr service unit count', type=int)
+        c.argument('sku', help='The sku name of the webpubsub service. E.g. Standard_S1')
```

Comment:
why diable the choice options of sku argument?

Reply:
We have SKUs that are internal only. Curretly we don't hope customers outside Microsoft to see those SKUs as they are not allowed to create them. But we hope az cli is available for our internal customers. So we need to remove this restriction from client side and leave it to server side.

Reply:
The sku parameter is required for `az webpubsub create`. Did you mean the customers outside Microsoft are not supposed to run the create command?

Reply:
I think we can put all `sku` available for public in `help`. But as it's a server-side parameter, I think it doesn't make sense to make restrictions in cli side. We can pass `sku` parameter to RP side for checking.

Reply:
> I think we can put all `sku` available for public in `help`. But as it's a server-side parameter, I think it doesn't make sense to make restrictions in cli side. We can pass `sku` parameter to RP side for checking.

I totally agree

Reply:
Fixed